### DISCUSSION TITLE
CSV Exporting dates as 1950-01-01 00:00:00.000 instead of mm/dd/yyyy

### DISCUSSION THREAD

**Discussion by PowerBIWiz** [DaxStudio/DaxStudio#804] [Q&A]
Dates appear in correct format in Grid view, but not the export file. Expected format is mm/dd/yyyy and that is shown in the grid, it's also the default format in the model. However in the CSV export file the date is not changing. I tried manually specifying the date format in the options and also clearing it in the options. Neither worked.

**Comment by dgosbell** [comment]
I've posted a response to this in the linked issue you created
